rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -464,9 +464,16 @@ rocalTensorList *
 MasterGraph::get_output_tensors()
 {
     auto output_ptr = _ring_buffer.get_read_buffers();
+    std::vector<unsigned* > roi_ptr = _ring_buffer.get_read_roi_buffers();
+    // TODO - check here if size of internal tensor and ring buffer is same?
+    auto deleter=[&](unsigned* ptr){ };
```

Comment:
Need to validate the correctness of this and check if this can be handled in a better way